src/routes/user.route.ts

```diff
@@ -0,0 +1,49 @@
+import { Request, Response, Router } from "express";
+
+import { userSchema } from "../models/user.model";
+import { createUser, getUserByEmail } from "../services/user.service";
+import { UserAlreadyExistsException } from "../utils/exceptions";
+import logger from "../utils/logger";
+
+const router = Router();
+
+router.post("/register", async (req: Request, res: Response) => {
+  try {
+    const { error } = userSchema.validate(req.body);
+    if (error) {
+      logger.error("Validation error: " + error.details[0].message);
```

Comment:
Always try to not duplicate code [DRY](https://www.techtarget.com/whatis/definition/DRY-principle),
If you use `error.details[0].message` twice, make a variable and reuse it.

Reply:
I've made an `errorMessage` constant to store error message.